api/src/utils/constants/misc.js

```diff
@@ -1,19 +1,11 @@
 exports.SYSTEM_USER = 'SYSTEM_USER';
 
-// Available user roles.
-exports.ROLES = {
-  SYSADMIN: 'sysadmin',
-  LNGADMIN: 'admin:lng',
-  NRCEDADMIN: 'admin:nrced',
-  BCMIADMIN: 'admin:bcmi',
-
-  ADMIN_ROLES: [
-    'sysadmin',
-    'admin:lng',
-    'admin:nrced',
-    'admin:bcmi'
-  ]
-};
+exports.ApplicationRoles = {
+  ADMIN: 'sysadmin',
+  ADMIN_NRCED: 'admin:nrced',
+  ADMIN_LNG: 'admin:lng',
```

Comment:
Need to add ADMIN_ROLES back or change everywhere it is used.

Reply:
I'll change how it works.

Reply:
Fixed in 59a3d4f